tests/unit/test_balancer.py

```diff
@@ -111,7 +112,7 @@ def test_client_ready(client, state):
         assert not client.ready
 
 
-def test_balancer_manager_create_internal_topics(harness):
+def test_balancer_manager_create_internal_topics(harness: Harness[KafkaCharm]):
```

Comment:
**praise:** I saw you doing the `Harness[KafkaCharm]` elsewhere too, really helps with the typing, thank you!